inc/reminder.class.php

```diff
@@ -1047,13 +1197,17 @@ static function showListForCentral($personal = true) {
             return false;
          }
 
-         $query = "SELECT `glpi_reminders`.*
-                   FROM `glpi_reminders`
-                   WHERE `glpi_reminders`.`users_id` = '$users_id'
-                         AND (`glpi_reminders`.`end` >= '$today'
-                              OR `glpi_reminders`.`is_planned` = 0)
-                         $restrict_visibility
-                   ORDER BY `glpi_reminders`.`name`";
+         $criteria = [
```

Comment:
en relation with previous comment, i think the code should be:
```php
$criteria = [
   'FROM'   => 'glpi_reminders',
   'WHERE'  => [
      'users_id'  => $users_id,
      [
         'OR' => [
            'end'          => ['>=', $today],
            'is_planned'   => 0
         ]
      ]
   ] + $visibility_criteria,
   'ORDER'  => 'name'
];
```

maybe with array_merge[_recursive]